Description: Remove the timestamp and set the locale when generating the javadoc to make it reproducible
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/build.xml
+++ b/build.xml
@@ -525,6 +525,8 @@
          windowtitle="${Name} ${version} API"
          doctitle="${Name} ${version} API"
          bottom="Copyright &amp;copy; ${year} The Apache Software Foundation"
+         locale="en"
+         additionalparam="-notimestamp"
          >
     	<packageset dir="${java.src.dir}">
           <include name="org/apache/**"/>
@@ -549,6 +551,8 @@
          windowtitle="${Name} ${version} API"
          doctitle="${Name} ${version} API"
          bottom="Copyright &amp;copy; ${year} The Apache Software Foundation"
+         locale="en"
+         additionalparam="-notimestamp"
          doclet="org.apache.yetus.audience.tools.IncludePublicAnnotationsStandardDoclet"
          docletpath="${ivy.lib}/audience-annotations-${audience-annotations.version}.jar"
          >
